src/containers/containerCollection/containerCollection.jsx

```diff
@@ -195,11 +195,13 @@ export class ContainerCollection extends Component {
 
     render () {
         const { classes } = this.props;
-
+        const totalSpaces = this.props.containers.reduce((total, container) => total + container.size, 0)
```

Comment:
Could you put all this logic in a function, so that it's neater.

Reply:
I agree with @deonjo